Source/Core/UICommon/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if(ENABLE_LLVM)
   if(LLVM_FOUND)
     message(STATUS "LLVM found, enabling LLVM support in disassembler")
     target_compile_definitions(uicommon PRIVATE HAVE_LLVM)
-    target_link_libraries(uicommon PRIVATE ${LLVM_AVAILABLE_LIBS})
+    target_link_libraries(uicommon PRIVATE LLVM)
```

Comment:
Scratch that, apparently the LLVMConfig thing is unconventional. Regardless, this change is wrong.